visma/calculus/differentiation.py

```diff
@@ -42,11 +43,11 @@ def differentiate(tokens, wrtVar):
     return tokens, availableOperations, token_string, animation, comments
 
 
-def differentiateTokens(funclist, wrtVar):
+def DifferentiateTokens(funcList, wrtVar):
```

Comment:
Only class names start with an uppercase. Since this is a function, the name must be `differentiateTokens`. Also let the name `workspace` remain as is. Other changes are fine.

Reply:
ahaa Fine got it. <3